### PR TITLE
Update LICENSE to Hippocratic 2.1

### LINKED ISSUE #53: Update LICENSE to Hippocratic 2.1
To discourage use of Lansing Codes' community-building software by anyone who would violate others' human rights, we are adopting the latest version of the ethical source license, Hippocratic 2.1.

In order to use the latest version of this license, the following changes are required:

1. Add or update the `LICENSE` file in this project to the [Hippocratic 2.1 Markdown](https://firstdonoharm.dev/version/2/1/license.md) text and make the following substitutions:
    - Replace `[SOFTWARE NAME]` with the GitHub organization and project name (e.g. `@lansingcodes/slackbot`)
    - Replace `(YEAR)` with `2015-Present`
    - Replace `(COPYRIGHT HOLDER(S)/AUTHOR(S))` with `Humanity Codes, LLC`
2. Add or update the `README.md` file in this project to include the following Markdown snippet:
    ``` md
    ## License
    
    [Hippocratic 2.1](https://firstdonoharm.dev)
    
    Copyright (c) 2015-Present, Humanity Codes LLC
    ```
3. If a `package.json` file exists in this project, add or update the `license` field in the file to look like this:
    ``` json
    "license": "Hippocratic-2.1",
    ```

---

**When submitting the pull request to resolve this issue, please assign @egillespie as the reviewer.**

### PR DESCRIPTION
**Pull request by dhcrain**
Resolves #53

### CODE REVIEW

**Comment by dhcrain** [comment]
The CI failure seems to be unrelated to the changes I made here.

File: LICENSE [hunk 1]
```diff
@@ -1,42 +1,33 @@
...
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
-FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
-COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
-IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
-CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
-This Hippocratic License is an Ethical Source license
-(https://ethicalsource.dev) derived from the MIT License, amended to limit the
-impact of the unethical use of open source software.
+@lansingcodes/slackbot Copyright 2020-Present (Humanity Codes, LLC)(“Licensor”)
```
Comment (egillespie) [inline LICENSE:1]: Do you mind updating the year to be `2015` here? That's when the slackbot was originally created.

File: README.md [hunk 2]
```diff
@@ -33,6 +33,6 @@ Before you get ahead of yourself, though, please read our
 
 ## License
 
-[MIT - Hippocratic](https://firstdonoharm.dev)
+[Hippocratic 2.1](https://firstdonoharm.dev)
 
-Copyright (c) 2015-present, Humanity Codes LLC
+Copyright (c) 2020-Present, Humanity Codes LLC
```
Comment (egillespie) [inline README.md:38]: Do you mind updating the year to be `2015` here? That's when the slackbot was originally created. 🙂

**Review by egillespie — CHANGES_REQUESTED** [review]
@dhcrain, it looks like the copyright year got changed to 2020 in two places. It should stay 2015 to match the creation of the project. Let me know when you've touched this up and I'll merge it. Thanks! 🙇

File: LICENSE (hunk 1 above)
Comment (dhcrain) [reply LICENSE]: Sorry, Copy and paste error.

File: README.md (hunk 2 above)
Comment (dhcrain) [reply README.md]: Sorry, Copy and paste error.

**Review by egillespie — APPROVED** [review]
Perfect. 👌

**Pull request merged**